.storybook/webpack.config.js

```diff
@@ -1,8 +1,13 @@
 /* eslint-disable no-param-reassign */
 const { resolve } = require('path');
 const autoprefixer = require('autoprefixer');
+const crypto = require('crypto');
+
+const cryptoCreateHash = crypto.createHash;
+crypto.createHash = (algorithm) => cryptoCreateHash(algorithm === 'md4' ? 'sha256' : algorithm);
```

Comment:
This is an ugly, band-aid fix to force Webpack 4 to use sha256 for its hash generation, since `md4` appears to be unsupported in OpenSSL 3.0 used by Node >17.